scripts/update-translations.sh

```diff
@@ -22,6 +22,14 @@ then
     abort "You must be on master to run this command."
 fi
 
+git fetch
+tracking_branch=`git rev-parse --abbrev-ref --symbolic-full-name @{u}`
+upstream_changes=`git rev-list HEAD...$tracking_branch --count`
```

Comment:
interesting, so this checks whether there are upstream changes on whatever branch you're on, but not that you're on branch master. Any particular reason for that?

Reply:
The previous check aborts if you are not on master.

Reply:
I did it this way in case anyone has the upstream remote not set called `origin`